resources/templates/md/pages/about.md

```diff
@@ -3,13 +3,20 @@
  :page-index 1
  :navbar? true}
 
-The Bangalore Clojure Group hosts one of the most active (since 2013) meetups for Clojure in India, each of our meetups are conducted in different locations with different set of speakers every month.Our meetups are scheduled conveniently on every first Saturday of the month.
+The Bangalore Clojure Group hosts one of the most active (since 2013) Clojure meetups in India. Each of our meetups are conducted at different locations with different set of speakers every month. Our meetups are scheduled conveniently on every first Saturday of the month.
 
-Our meetups usually have talks by clojurists, but we also do organise hackathons on a regular basis. We are an open community, we encourage all Clojurists & functional freaks without any bais (We also welcome windows users 🙂)
+Our meetups usually have talks by clojurists, but we organise hackathons on regular basis as well. We are an open community, we encourage all Clojurists & functional freaks without any bais (We also welcome windows users 🙂)
```

Comment:
Consider making `c` in `clojursists` upper case for standardisation and consistency. Also, consider replacing `&` with `and`. And, s/bais/bias. Make `w` in `windows` upper case and end the sentence with a period.

Reply:
fixed